Library/Homebrew/formula_creator.rb

```diff
@@ -9,11 +9,21 @@ module Homebrew
   #
   # @api private
   class FormulaCreator
-    attr_reader :args, :url, :sha256, :desc, :homepage
-    attr_accessor :name, :version, :tap, :mode, :license
-
-    def initialize(args)
-      @args = args
+    attr_reader :url, :tap, :sha256, :desc, :homepage
+    attr_accessor :name
+
+    sig {
+      params(name: T.nilable(String), version: T.nilable(String), tap: T.nilable(String), mode: T.nilable(Symbol),
+             license: T.nilable(String), fetch: T::Boolean, head: T::Boolean).void
+    }
+    def initialize(name, version, tap:, mode:, license:, fetch: true, head: false)
```

Comment:
```suggestion
    def initialize(name, version, tap:, url:, mode:, license:, fetch:, head:)
```

unless there's callers I missed where these aren't passed